.github/workflows/ci.yml

```diff
@@ -76,7 +76,6 @@ jobs:
           CHROMIUM: true
         run: |
           xvfb-run --auto-servernum npm run test:unit:coverage
-          xvfb-run --auto-servernum npm run test:coverage
```

Comment:
I think the previous version would throw an error if something is not covered by tests? Can we have something similar in CI still?

Reply:
also does not it cause failures because c8 instruments the code that we run in the page context via CDP?